UI/Test/tests_switch_language.py

```diff
@@ -0,0 +1,25 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from configuration import URL
+from configuration import driver
+
+
+def test_open_epam_and_check_element(driver):
+    driver.get(URL)
```

Comment:
The same, you can add fixture for it and reuse

Reply:
I didn't do it because there are different URLs in other tests. It is more convenient to keep it as is.